sdk/identity/identity/test/internal/node/workloadIdentityCredential.spec.ts

```diff
@@ -72,7 +72,6 @@ describe("WorkloadIdentityCredential", function () {
   it("authenticates with ManagedIdentity Credential", async function (this: Context) {
     process.env.AZURE_FEDERATED_TOKEN_FILE = tokenFilePath;
     const credential = new ManagedIdentityCredential("dummy-clientId");
-    assert.equal(credential["clientId"], "dummy-clientId");
```

Comment:
We shouldn't be testing private fields if we can avoid it. This doesn't seem to add any value - the test should be "we can authenticate with a client ID" as opposed to "we set a private field correctly"